packages/components/collapse/Collapse.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Collapse'
+slug: /components/collapse/
+github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/collapse'
+storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-collapse'
+typescript: ./src/Collapse.tsx,
+v4Doc: true
+---
+
+import { SectionHeading, Stack, Button } from '@contentful/f36-components';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { PropsHeading } from '@contentful/f36-docs-utils';
+
+Collapse is used to elegantly hide and show extended content programmatically.
+
+## Table of contents
+
+- [Import](#import)
+- [Examples](#examples)
+  - [Basic usage](#basic-usage)
+- [Props](#props-api-reference)
+- [Content guidelines](#content-guidelines)
+- [Accessibility](#accessibility)
+- [Help improve this page](#help-improve-this-page)
+
+## Import
+
+```jsx static=true
+import { Collapse } from '@contentful/f36-components';
+```
+
+## Examples
+
+The collapse component is a basic component to show and hide content programmatically.
+
+### Basic usage
+
+```jsx
+function CollapseExample() {
+  const [isExpanded, setIsExpanded] = React.useState(true);
+  return (
+    <div>
+      <Button onClick={() => setIsExpanded(!isExpanded)}>Toggle</Button>
+      <Collapse isExpanded={isExpanded}>
+        <Text>
+          Customers on the Team tier can pay with a credit card (American
+          Express, MasterCard or Visa). Enterprise customers have the choice of
+          paying with a credit card or wire transfer.
+        </Text>
+      </Collapse>
+    </div>
+  );
+}
+```
+
+### Collapse
+
+<PropsHeading of="Collapse" />
```

Comment:
Oh sorry I should have explained better

PropsHeading is the component only for the title of this section, it's not the list of props. 
You still need to use `Props`, this actually gives me the idea of merging both components 🤔 
Check how it's done in the accordion now: https://github.com/contentful/forma-36/blob/forma-v4/packages/components/accordion/Accordion.mdx

```
<PropsHeading storybookPath="/story/components-accordion" />

### Accordion

<Props of="Accordion" />
```

I guess in your case it will be something like:

```suggestion
<PropsHeading storybookPath="/story/animation-collapse" />

<Props of="Collapse" />
```

ping me if you need any help